### ISSUE TITLE
Feature Request: Set-SecretInfo should take pipeline input

### ISSUE THREAD

**Issue by jdhitsolutions** [PowerShell/SecretManagement#129]
It is a common PowerShell practice to run a Get command and pipe to a Set command. I would expect to be able to run a command like this:

```powershell
get-secretinfo -Vault $vault -Name foo* | Set-SecretInfo -Metadata $meta
```

But the Name and Vault parameters on Set-SecretInfo don't accept pipeline input. I can't imagine why they shouldn't.

**Comment by PaulHigin** [comment]
Yes, this support can be added.

**Comment by PaulHigin** [comment]
Implemented in v1.1.0-preview
